tools/generator/src/Generator+SetTargetConfigurations.swift

```diff
@@ -106,13 +110,13 @@ Target "\(id)" not found in `pbxTargets`
                 )
             }
 
-            if !target.links.isEmpty {
-                let linkFileList = filePathResolver
+            if !target.linkerInputs.staticLibraries.isEmpty {
+                let linkFileList = try filePathResolver
                     .resolve(try target.linkFileListFilePath())
                     .string
                 try targetBuildSettings.prepend(
                     onKey: "OTHER_LDFLAGS",
-                    ["-filelist", "\(linkFileList),$(BUILD_DIR)".quoted]
+                    ["-filelist", linkFileList.quoted]
```

Comment:
This change was to setup support for `cc_import` and `objc_import`. Both of those will have `PROJECT_DIR` relative paths.